File: Battle of the characters (Easy).js
function battle(x, y) {
  const alpha = "abcdefghijklmnopqrstuvwxyz".toUpperCase()
  const x_v = x.split("").map(v => alpha.indexOf(v)+1).reduce((a,b) => a+b,0)
  const y_v = y.split("").map(v => alpha.indexOf(v)+1).reduce((a,b) => a+b,0)
  if(x_v===y_v){
    return "Tie!"
  }else if(x_v>y_v){
    return x
  }else{
    return y
  }
}

File: Battle of the characters (Easy).py
def battle(x, y):
    x_v = sum([ord(v)-64 for v in list(x)])
    y_v = sum([ord(v)-64 for v in list(y)])
    if x_v==y_v:
        return "Tie!"
    elif x_v>y_v:
        return x
    else:
        return y
    
